scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -616,15 +624,15 @@ function MetaTermsDirective(metadata, $filter, $timeout) {
                     var o = {};
 
                     // dependent is set only for category
-                    if (scope.dependent) {
+                    if (scope.resetDependent) {
```

Comment:
I don't think this will ever resolve to `true`. Even if the function is not passed it will have a wrapper. You may need to use `iAttrs` attribute to check if it is there.